dataretrieval/utils.py

```diff
@@ -161,6 +163,16 @@ def set_metadata(response):
     return md
 
 
+def set_useragent():
```

Comment:
I think that makes sense. I hadn't gone that way because:

1. Modifying `__init__.py` isn't recommended by `setuptools_scm`
2. I couldn't think of any other instance where we'd need to access the version number (since the documentation case is covered in a different way per `setuptools_scm` documentation on best way to do that). 

So I guess with those two items in mind, do you think it's helpful to have the version information accessible at the package level in `__init__.py`? I'm flexible, so happy to make this change if you think it's helpful.